scripts/run.py

```diff
@@ -97,9 +97,9 @@
 for neuron in neurons:
     # Construct the PM2 start command
     command = (
-        f"pm2 start {neuron['file']} --interpreter python3 --name {neuron['hotkey']}:{neuron['type']} --"
-        + f" --wallet.name {coldkey} --wallet.hotkey {neuron['hotkey']} --subtensor.network {network} --netuid {netuid}"
-        + f" --axon.port {neuron['port']} --logging.debug {neuron.get('config')}"
+        f"pm2 start {neuron["file"]} --interpreter python3 --name {neuron["hotkey"]}:{neuron["type"]} --"
+        + f" --wallet.name {coldkey} --wallet.hotkey {neuron["hotkey"]} --subtensor.network {network} --netuid {netuid}"
+        + f" --axon.port {neuron["port"]} --logging.debug {neuron.get("config")}"
```

Comment:
This code will result in error with our target python versions 3.9, 3.10

Reply:
I'll Revert this part

Reply:
@HrushiYadav thanks! Also make sure to make the change against pre-staging branch, or even: https://github.com/macrocosm-os/prompting/tree/SN1-150-remove-low-value-subcompetitions

It's a new upcoming change with lots of changes and refactor done

Reply:
I'll commit to same branch